crashsimilarity/utils.py

```diff
@@ -17,6 +17,37 @@ def read_files(file_names, open_file_function=smart_open):
                 yield line if isinstance(line, str) else line.decode('utf8')
 
 
+class StackTraceProcessor(object):  # just a namespace, actually
+    @staticmethod
+    def _should_skip(stack_trace):
```

Comment:
They are used in other modules for backward compatibility. I will make them public if they will be needed in new code. In Algorithms or Models, for example.